Makefile

```diff
@@ -0,0 +1,29 @@
+OPENTELEMETRY_CHART = ./charts/opentelemetry-collector
+EXAMPLES_DIRECTORY = ./examples/opentelemetry-collector
```

Comment:
I like the word `examples` as the outer folder since I think it makes them easier to find.  How about `./charts/opentelemetry-collector/examples` but in each subfolder instead of `manifest` we use `rendered` ?

Reply:
Sounds good to me